test.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+	class sampleClass (){

```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Class name "sampleClass" is not in camel caps format